pennylane/devices/default_mixed.py

```diff
@@ -181,9 +221,17 @@ def _apply_channel(self, kraus, wires):
         kraus_dagger = [self._conj(self._transpose(k)) for k in kraus]
 
         # Changes tensor shape
-        kraus_shape = [len(kraus)] + [2] * num_ch_wires * 2
-        kraus = self._cast(self._reshape(kraus, kraus_shape), dtype=self.C_DTYPE)
-        kraus_dagger = self._cast(self._reshape(kraus_dagger, kraus_shape), dtype=self.C_DTYPE)
+        if kraus[0].shape[0] == kraus[0].shape[1]:
+            kraus_shape = [len(kraus)] + [2] * num_ch_wires * 2
+            kraus = self._cast(self._reshape(kraus, kraus_shape), dtype=self.C_DTYPE)
+            kraus_dagger = self._cast(self._reshape(kraus_dagger, kraus_shape), dtype=self.C_DTYPE)
+        elif (kraus[0].shape == (1, 2)) and (num_ch_wires == 1):
+            kraus_shape = [len(kraus)] + list(kraus[0].shape)
+            kraus = self._cast(self._reshape(kraus, kraus_shape), dtype=self.C_DTYPE)
+            kraus_dagger_shape = [len(kraus)] + list(kraus[0].shape)[::-1] * num_ch_wires
```

Comment:
As per the condition, `(num_ch_wires == 1) == True` stands
```suggestion
            kraus_dagger_shape = [len(kraus)] + list(kraus[0].shape)[::-1]
```